### DISCUSSION TITLE
Cannot edit pages created from templates

### DISCUSSION THREAD

**Discussion by andromika** [requarks/wiki#3772] [Error / Bug Report]
After creating a page from a template and attempting to edit it, you'll be prompted to create a new page, which if you try to save will return an error saying the page is already in use.

In order to actually edit the page, you have to tell the "create new page" prompt that you want to create it from a template. pick any random page as a template and instead of a new page you'll be redirected to the edit mode of the original page you intended to edit.

Running **2.5.144**

**EDIT:**
Trying to edit the page in that state doesn't work either. Your changes are immedately discarded upon saving.
Deleting the page and re-creating it just restores it to the previous version.

Attempting to move it will permanently leave it stuck and it can't be deleted/moved/edited anymore with the error "Invalid page ID"

**EDIT 2**
Behavior persists in 2.5.197

**Comment by andromika** [comment]
bug persists in 2.5.201

**Comment by dnjackson** [comment]
I came across this because I'm trying to figure out how to create a new page from a template at all. I see no way of doing this. Has the feature been removed from wiki.js, perhaps because of this bug?
